instance/restage.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2017-Present Pivotal Software, Inc. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the under the Apache License, Version 2.0 (the "License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package instance
+
+import (
+	"fmt"
+
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient"
+)
+
+func Restage(authenticatedClient httpclient.AuthenticatedClient, serviceInstanceAdminURL string, accessToken string) (string, error) {
+	_, err := authenticatedClient.DoAuthenticatedPut(fmt.Sprintf("%s/command?restage", serviceInstanceAdminURL), accessToken)
+	return "", err
+}
```

Comment:
There is no automated test code for this new functionality. Same comment applies for the recently added `start.go` and `restart.go` as well. Please could you add some test code for the restaging operation in a new commit. If you want to add the start and restart tests in the same commit then that's fine, otherwise they could be covered in a separate story. Thanks.

Reply:
Fixed !